pantheon/src/main/java/tech/pegasys/pantheon/controller/IbftPantheonController.java

```diff
@@ -214,21 +215,20 @@
 
     final IbftGossip gossiper = new IbftGossip(uniqueMessageMulticaster);
 
+    final ScheduledExecutorService timerExecutor = Executors.newSingleThreadScheduledExecutor();
```

Comment:
Could be good idea to use the MonitoredExecutors.newFixedThreadPool so we can give our thread pool a name and get some free metrics as well.

Reply:
done.